webapp/components/FilterPosts/GeneralFilterMenuItems.vue

```diff
@@ -13,17 +13,18 @@
           <ds-flex-item width="10%" />
           <ds-space margin-bottom="xx-small" />
           <ds-flex-item width="100%">
-            <div class="follow-button">
-              <ds-button
+            <div class="follow-filter-button">
+              <base-button
+                data-test="filter-by-followed"
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-01-14T17:22:50Z" title="Tuesday, January 14th 2020, 6:22:50 pm +01:00">Jan 14, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -13,17 +13,18 @@
           <ds-flex-item width="10%" />
           <ds-space margin-bottom="xx-small" />
           <ds-flex-item width="100%">
-            <div class="follow-button">
-              <ds-button
+            <div class="follow-filter-button">
+              <base-button
+                name="filter-by-followed-authors-only"
```


I know you didn't add this, but is this here only for testing purposes?
if so, wouldn't it be better to use `data-test`, which is removed in production?

Reply:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2020-01-15T07:44:20Z" title="Wednesday, January 15th 2020, 8:44:20 am +01:00">Jan 15, 2020</time>_

---


ha, I'll go check!